src/components/index.js

```diff
@@ -1,8 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+
 export { default as Navigation } from "./Navigation";
 export { default as Footer } from "./Footer";
 export { default as Login } from "./Login";
 export { default as Home } from "./Home";
 export { default as About } from "./About";
 export { default as NotFound } from "./NotFound";
 export { default as Profile } from "./Profile/Profile";
-export { default as UserDashboard } from "./UserDashboard";
\ No newline at end of file
+export { default as UserDashboard } from "./UserDashboard";
```

Comment:
This is not correct, this file is only for exporting and not rendering

Reply:
reverted